### PR TITLE
chore(context)!: something

### PR DESCRIPTION
**Pull request by dvdagreat**
add config file

BREAKING CHANGE: removing old teachers api breaks student api refs:DOCS-1234

BE-2

### CODE REVIEW

**Comment by PandyaSmit** [comment]
please optimize the code @dvdagreat

File: maja
Comment (iamchiraggiri) [inline maja:1]: Hov hov maja ma

**Review by iamchiraggiri — CHANGES_REQUESTED** [review]

**Comment by iamchiraggiri** [comment]
****

**Review by PandyaSmit — COMMENTED** [review]
NOICE

**Review by iamchiraggiri — APPROVED** [review]
Mai ja

**Review by PandyaSmit — APPROVED** [review]
DONE DONA DONE

**Pull request merged**